tools/lsp/ui/main.slint

```diff
@@ -142,4 +144,9 @@ export component PreviewUi inherits Window {
         x: (parent.width - self.width) / 2;
         y: (parent.height / 10);
     }
+
+    if show-floating-widget: ColorPickerView {
```

Comment:
Another nit: `show-floating-widget` seems a rather generic name for something that controls the `ColorPickerView`.